tests/test_updater_dep_install.py

```
@@ -0,0 +1,166 @@
+
+    captured = {}
+
+    async def fake_run(cmd, cwd=None, timeout=600.0, env=None):
+        captured["cmd"] = cmd
+        captured["cwd"] = cwd
+        captured["env"] = env
+        return 0, "installed"
+
+    monkeypatch.setattr(settings_mod, "_run_capture", fake_run)
+
+    rc, out = await settings_mod._install_dependencies(project)
```

Ruff (0.15.20)

[warning] Unpacked variable `out` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)